index.html

```diff
@@ -1,256 +1,272 @@
+        <h3>Skills</h3>
+        <ul>
+          <li>Codekit</li>
+          <li>Codepen</li>
+          <li>GitHub</li>
+          <li>GitLAb</li>
+          <li>Terminal</li>
         </ul>
-      </section>
-      <section class="contact-form">
-        <div class="contact-heading">
-            <h1 class="heading-1">
-                Get Started
-            </h1>
-            <p class="par-1">Lorem ipsum dolor sit amet consectetur adipisicing elit.
-             Unde ducimus corporis asperiores eveniet architecto, eos laborum velit quasi.
-             Debitis corporis provident cum nihil officiis recusandae explicabo consequuntur suscipit,
-             reiciendis enim!
-            </p>
-        </div> 
-        <div class="form-container">
-            <form action="https://formspree.io/f/xwkanyro" method="POST" class="form">
-          
-              <input type="text" id="fname" name="fullname" placeholder="Your name.." class="input-field" maxlength="30" required>
-           
-              <input type="email" id="email" name="email" placeholder="email@address.com" class="input-field" required>
-
-              <textarea id="subject" name="subject" placeholder="Write something.." class="text-area" maxlength="500" required></textarea>
-
-              <button class="contact-button input-field" type="submit">Start Collaboration</button>
-            </form>
-          </div>   
-      </section>
-    </main>
-    <footer class="footer">
-    <span class="footer-btn">Get My resume<span class="footer-img"><img src="img/Union.png" alt=""></span></span>
-    <ul class="icons">
-        <li><ion-icon name="logo-stackoverflow"></ion-icon></li>
-        <li><ion-icon name="logo-github"></ion-icon></li>
-        <li><ion-icon name="logo-twitter"></ion-icon></li>
-        <li><ion-icon name="logo-linkedin"></ion-icon></li>
-        <li><ion-icon name="logo-medium"></ion-icon></li>
-        <li><ion-icon name="logo-angular"></ion-icon></li>
-    </ul>    
-</footer>
-    <script
-      type="module"
-      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"
-    ></script>
-  </body>
+        <img src="./Media/ic_skills.svg" alt=" " class="about-img">
+      </div>
+      <img class="deco-7" src="./Media/image_dots_with_teal.svg" alt=" ">
+    </div>
+  </section>
+  <section id="contact" class="contact-me">
+    <h2>Get started</h2>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+      tempor incididunt ut labore et dolore magna.
+    </p>
+    <form action="https://formspree.io/f/xwkanyro" method="POST">
+      <label>
+        <input type="text" maxlength="30" placeholder=" Your name" name="name" required>
+      </label>
+      <label>
+        <input type="email" placeholder=" Your email" name="email" required>
+      </label>
+      <label class="txt-area">
+        <textarea placeholder="Enter your message here..."  maxlength="500" required></textarea>
+      </label>
```

Comment:
- [ ] This section doesn't seem to like the given design. I think you forget to give a text to the label

Reply:
Thank you for the feedback I have added labels